components/DetailPageHeader.js

```diff
@@ -152,6 +162,12 @@ class DetailPageHeader extends Component {
               {' '}
               <span className="bx--detail-page-header-status-text">
                 {statusText}{statusContent}
+                <span className="bx--detail-page-header-tag">
+                  {' '}{tag}
```

Comment:
Not sure if this is the preferred way to do this, but I was unsure of how else to add a bit of spacing without CSS.